money_bot/initialization.py

```diff
@@ -0,0 +1,17 @@
+import logging
+
+from aiogram import Bot, Dispatcher
+from pymongo import MongoClient
+from umongo import Instance
+
+try:
+    from money_bot.local_config import BOT_TOKEN, DB_HOST, DB_NAME
+except ImportError:
+    from money_bot.example_config import BOT_TOKEN, DB_HOST, DB_NAME
```

Comment:
Read about imports in [Google Python style guide](https://github.com/google/styleguide/blob/gh-pages/pyguide.md#22-imports). General recommendations: avoid using `from smth import *` and prefer `from smth import package` to `from smth.package import VAR1, VAR2`

In this particular case, I recommend you to rewrite it with 

```python3
try:
    from money_bot import local_config as config
except ImportError:
    from money_bot import example_config as config
```
And then use the vars with `config.BOT_TOKEN` style. It will be much prettier to see and easier to maintain

Reply:
It's really better, thanks